Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/IssueList.hs

```diff
@@ -101,3 +103,16 @@ buildIssueList (issues, person) = do
         createdAt = issues.createdAt,
         updatedAt = issues.updatedAt
       }
+
+ticketStatusCallBack :: ShortId DM.Merchant -> Common.TicketStatusCallBackReq -> Flow APISuccess
+ticketStatusCallBack _ req = do
+  isPresent <- QIssue.isTicketPresent req.ticketId
+  unless isPresent $ throwError $ TicketDoesNotExist req.ticketId
```

Comment:
You can just use `findByTicketId` and throw error if its `Nothing` on the same line instead of creating an extra function